libnetwork/sandbox_dns_unix.go

```diff
@@ -172,7 +172,9 @@ func (sb *sandbox) setExternalResolvers(content []byte, addrType int, checkLoopb
 	for _, ip := range servers {
 		hostLoopback := false
 		if checkLoopback {
-			hostLoopback = dns.IsIPv4Localhost(ip)
+			if p := net.ParseIP(ip); p != nil {
```

Comment:
Isn't this different since it could be an ipv6 loopback?

Reply:
Ah, yes. Brainfart; I was thinking this was to _skip_ loopback IP-addresses (as they won't work as DNS for container), but this is the bit where we forward requests to them. Yup, that shouldn't use IPv6 likely. Or maybe it should, but we can do that separately.